src/docfx/lib/HtmlUtility.cs

```diff
@@ -395,34 +402,52 @@ private static bool IsLink(ref HtmlToken token, in HtmlAttribute attribute)
         private static int CountWordInText(ReadOnlySpan<char> text)
         {
             var total = 0;
-            var word = false;
+            var isEnglishWord = false;
```

Comment:
Technically this is not limited to `English`. All non CJK words fall under this category, so keep the existing name?

```suggestion
            var word = false;
```